ocs_sample_library_preview/DataView/SummaryDirection.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class SummaryDirection(Enum):
+    """
+    enum 1-2  fully inclusive
```

Comment:
should this enum be 0-1 instead of 1-2?

Reply:
Nope, there is actually no 0 value for this enum (I even checked internally)